gradoop-flink/src/main/java/org/gradoop/flink/model/api/epgm/BaseGraphCollectionOperators.java

```diff
@@ -89,4 +343,55 @@ default GC verifyGraphsContainment() {
   default GC apply(ApplicableUnaryBaseGraphToBaseGraphOperator<GC> operator) {
     return callForCollection(operator);
   }
+
+  /**
+   * Transforms a graph collection into a base graph by applying a
+   * {@link org.gradoop.flink.model.api.operators.BinaryBaseGraphToBaseGraphOperator}
```

Comment:
The method accepts a `ReducibleBinaryBaseGraphToBaseGraphOperator`, not just a `BinaryBaseGraphToBaseGraphOperator`